frontend/src/index.css

```diff
@@ -6,6 +6,15 @@ h3 {
   padding: 1rem 0;
 }
 
+h1 {
```

Comment:
I feel it's not a good practice to override the default size and padding of the elements directly as it might seem confusing when you back to work again on the project after some months and make a new h1 and find its font-size is not normal

Reply:
Yes you are right i will notice this point  later